bindings/mysql/mysql.go

```diff
@@ -147,7 +145,7 @@ func (m *Mysql) Invoke(ctx context.Context, req *bindings.InvokeRequest) (*bindi
 		return nil, errors.Errorf("required metadata not set: %s", commandSQLKey)
 	}
 
-	startTime := time.Now().UTC()
+	startTime := time.Now()
```

Comment:
Arguably returning the time in a local time now is a breaking change. Why not keep it UTC?